pkg/proxy/ipvs/proxier.go

```diff
@@ -191,8 +191,8 @@ var ipsetWithIptablesChain = []struct {
 	{kubeNodePortSetTCP, string(KubeNodePortChain), string(KubeMarkMasqChain), "dst", utilipset.ProtocolTCP},
 	{kubeNodePortLocalSetUDP, string(KubeNodePortChain), "RETURN", "dst", utilipset.ProtocolUDP},
 	{kubeNodePortSetUDP, string(KubeNodePortChain), string(KubeMarkMasqChain), "dst", utilipset.ProtocolUDP},
-	{kubeNodePortSetSCTP, string(KubeNodePortChain), string(KubeMarkMasqChain), "dst,dst", utilipset.ProtocolSCTP},
 	{kubeNodePortLocalSetSCTP, string(KubeNodePortChain), "RETURN", "dst,dst", utilipset.ProtocolSCTP},
+	{kubeNodePortSetSCTP, string(KubeNodePortChain), string(KubeMarkMasqChain), "dst,dst", utilipset.ProtocolSCTP},
```

Comment:
I'm not familiar with the ipset implementation, why sctp `matchType` is `dst,dst` but udp and tcp ar eonly `dst`?

Reply:
It's because `bitmap:port` ipset doesn't support matching SCTP port: https://github.com/kubernetes/kubernetes/pull/74341
https://ipset.netfilter.org/ipset.man.html#lbAV:
> The set match and SET target netfilter kernel modules interpret the stored numbers as TCP or UDP port numbers.